CONTRIBUTING.md

```diff
@@ -35,7 +35,7 @@ Many of the bicep integration tests rely on baseline test assertion files that a
 #### Manually
 * If you see a test failure with a message containing Windows and *nix copy commands, you have encountered such a test. You have the following options to fix the test:
   1. Manually execute the provided command in a shell. This makes sense for a single test, but is extremely tedious otherwise.
-  1. Run the `SetBaseline.ps1` script at the repo root to execute the tests in `SetBaseLine` mode, which causes the baselines to be automatically updated in bulk for failing tests. You should see baseline file modifications in Git pending changes. (Make sure your Git pending changes are empty before doing so - your changes could get overwritten!).
+  1. Run the `SetBaseline.ps1` script in [scripts](./scripts) to execute the tests in `SetBaseLine` mode, which causes the baselines to be automatically updated in bulk for failing tests. You should see baseline file modifications in Git pending changes. (Make sure your Git pending changes are empty before doing so - your changes could get overwritten!).
```

Comment:
we could add ``SetBaseline.ps1/`SetBaseline.sh`

Reply:
Good point!